<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>facultyForm</class>
 <widget class="QDialog" name="facultyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_facultyInfo">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Faculty Card Information:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_position">
       <property name="text">
        <string>Position:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_position</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_position"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_collegeEmployed">
       <property name="text">
        <string>College Employed:</string>
       </property>
       <property name="buddy">
        <cstring>comboBox_facultyColleges</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_facultyColleges">
       <item>
        <property name="text">
         <string>- Please Select -</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>U.A. Whitaker College of Engineering</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>College of Arts and Sciences</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lutgert College of Business</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>College of Education</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Marieb College of Health &amp; Human Services</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other - Please Specify</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_facultyOtherCollege">
       <property name="text">
        <string>Other College:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_facultyOtherCollege</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_facultyOtherCollege"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_yearsEmployed">
       <property name="text">
        <string>Year Employed:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_yearsEmployed</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_yearsEmployed"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_create">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
